go/libkb/json.go

```diff
@@ -306,6 +306,17 @@ func (f *jsonFileTransaction) Abort() error {
 	return err
 }
 
+// Rollback reloads config from unchanged config file, bringing its
+// state back to from before the transaction changes. Note that it
+// only works for changes that do not affect UserConfig, which caches
+// values, and has to be reloaded manually.
+func (f *jsonFileTransaction) Rollback() error {
```

Comment:
does this work if there was no config file before?

Reply:
fixed in latest commit, can you take a look again? thank you!